### ISSUE TITLE
Dijkstra SSSP algo returns error

### ISSUE THREAD

**Issue by tomasonjo** [neo4j-contrib/neo4j-graph-algorithms#276]
I am trying out algo on the example graph from the docs.
I am using algos 3.1.5.1 on Neo4j 3.1.6.
_I also build the current version of algos and it return the same error_

```
CREATE (a:Loc{name:'A'}), (b:Loc{name:'B'}), (c:Loc{name:'C'}), 
       (d:Loc{name:'D'}), (e:Loc{name:'E'}), (f:Loc{name:'F'}),
       (a)-[:ROAD {cost:50}]->(b),
       (a)-[:ROAD {cost:50}]->(c),
       (a)-[:ROAD {cost:100}]->(d),
       (a)-[:RAIL {cost:50}]->(d),
       (b)-[:ROAD {cost:40}]->(d),
       (c)-[:ROAD {cost:40}]->(d),
       (c)-[:ROAD {cost:80}]->(e),
       (d)-[:ROAD {cost:30}]->(e),
       (d)-[:ROAD {cost:80}]->(f),
       (e)-[:ROAD {cost:40}]->(f),
       (e)-[:RAIL {cost:20}]->(f);
```

When i run the dijkstra SSSP i get back an error:

query:

```
MATCH (start:Loc{name:'A'}), (end:Loc{name:'F'})
CALL algo.shortestPath.stream(start, end, 'cost') 
YIELD nodeId, cost
RETURN nodeId, cost LIMIT 20
```
Error:

> Failed to invoke procedure `algo.shortestPath`: Caused by: java.lang.NoSuchMethodError: org.neo4j.kernel.impl.util.collection.SimpleBitSet.clear()V

I get the same error both for writing back and streaming results.

**Comment by jexp** [comment]
Can you build from source? 
I had to copy code into the project b/c Neo4j removed the method.

**Comment by tomasonjo** [comment]
I cloned the original algo repo and builded the current version for 3.1.x and the error persists in neo4j 3.1.6

**Comment by jexp** [comment]
Could be that the SPI was changed before 3.1.6. I'll check.

**Comment by jexp** [comment]
Yes they changed the SPI :(

**Comment by jexp** [comment]
Pushed fix commit for 3.1
